addon-test-support/@ember/test-helpers/dom/focus.ts

```diff
@@ -28,7 +28,7 @@ export function __focus__(element: HTMLElement | Element | Document | SVGElement
     document.activeElement !== element &&
     isFocusable(document.activeElement)
   ) {
-    __blur__(document.activeElement);
+    __blur__(document.activeElement, element);
   }
```

Comment:
Based on https://github.com/emberjs/ember-test-helpers/pull/965/files#r544655601. This would become:

```js
if (
  document.activeElement &&
  document.activeElement !== element &&
  isFocusable(document.activeElement) &&
  browserIsNotFocused
) {
  __blur__(document.activeElement, element);
}
```

Reply:
Updated.